bundle/manifests/numaresources-operator.clusterserviceversion.yaml

```diff
@@ -8,20 +8,28 @@ metadata:
           "apiVersion": "nodetopology.openshift.io/v1alpha1",
           "kind": "NUMAResourcesOperator",
           "metadata": {
-            "name": "numaresourcesoperator-sample"
+            "name": "numaresourcesoperator"
           },
           "spec": {
-            "foo": "bar"
+            "nodeGroups": [
+              {
+                "machineConfigPoolSelector": {
+                  "matchLabels": {
+                    "pools.operator.machineconfiguration.openshift.io/worker": ""
+                  }
+                }
+              }
+            ]
           }
         },
         {
           "apiVersion": "nodetopology.openshift.io/v1alpha1",
           "kind": "NUMAResourcesScheduler",
           "metadata": {
-            "name": "numaresourcesscheduler-sample"
+            "name": "numaresourcesscheduler"
           },
           "spec": {
-            "foo": "bar"
+            "imageSpec": "quay.io/openshift-kni/scheduler-plugins:4.10-snapshot"
```

Comment:
probably should be 4.11-snapshot because we already branched, and the release-4.10 should have 4.10-snapshot

Reply:
right, fixing. This however will mean we cannot easily cherry-pick the change. No big deal though.